Sming/third-party/.patches/axtls-8266.patch

```diff
@@ -196,6 +222,24 @@ index 0cf2e4c..78576f7 100644
      if (IS_SET_SSL_FLAG(SSL_SENT_CLOSE_NOTIFY))
          return SSL_CLOSE_NOTIFY;
  
+@@ -2226,8 +2229,15 @@ void DISPLAY_STATE(SSL *ssl, int is_send, uint8_t state, int not_ok)
+     if (!IS_SET_SSL_FLAG(SSL_DISPLAY_STATES))
+         return;
+ 
+-    printf(not_ok ? "Error - invalid State:\t" : "State:\t");
```

Comment:
For that line: can you just change it from:

```
printf(not_ok ? "Error - invalid State:\t" : "State:\t");
```
to

```
printf("%s", (not_ok ? "Error - invalid State:\t" : "State:\t") );
```

That change does not add new lines and will be easier to merge into `upstream` or other axtls clones.